tools/openshift-ci/Dockerfile

```diff
@@ -36,5 +36,5 @@ RUN set -x && \
 
 # Fixing OCPQE-11756
 RUN set -x && \
-    mkdir /verification-tests/.local && \
-    chmod -R 777 /verification-tests/.local
+    mkdir /.local && \
```

Comment:
I'm sorry, where are we creating a `local` directory with command `mkdir /.local`?

Reply:
In the root folder `/`